profiles/urls.py

```diff
@@ -1,6 +1,11 @@
 from django.contrib import admin
 from django.urls import path
 
-urlpatterns = [
+from profiles.views import StartUpProfilesView, StartUpProfileCreate, StartUpProfileUpdate, StartupProfileViewById
 
+urlpatterns = [
+    path('startups/', StartUpProfilesView.as_view(), name='startup-list'),
+    path('create', StartUpProfileCreate.as_view(), name='startup-create'),
+    path('update/<int:pk>/', StartUpProfileUpdate.as_view(), name='star-update'),
```

Comment:
The URL pattern name `star-update` is incomplete. It should probably be `startup-update` for consistency.